guide.cue

```diff
@@ -14,6 +14,7 @@ _#installGo:          "playwithgo/installgo1.15.8@sha256:0e480b658f50b85b8eb40c4
 _#go115LatestVersion: "go1.15.8"
 _#go115LatestImage:   "playwithgo/go1.15.8@sha256:7640da09d1555c4dddbba7f1b96051af2816e6542005176b749f38865ee0454c"
 _#go116LatestImage:   "playwithgo/go1.16@sha256:3ae1950433998a2be8c8ce3b1cb6479e6541f1e32443447a24085cfe09e2c391"
+_#go117LatestImage:   "tpaschalis-gotip"
```

Comment:
I've tried to bump the `gotip` image to `go1.17beta1` but with little success due to some Ubuntu package dependency weirdness [here](https://github.com/play-with-go/docker/pull/33)

For now, I created my own 1.17 image without Git, maybe we should fix that issue first.

Reply:
Just fixed this here: https://github.com/play-with-go/docker/commit/41edcfa7b93617b276de0cd59853620ef96e34f0